docs/content/spark/procedures.md

```diff
@@ -101,6 +101,18 @@ This section introduce all available spark procedures about paimon.
           CALL sys.rollback(table => 'default.T', version => 10)
       </td>
     </tr>
+    <tr>
+      <td>migrate_table</td>
+      <td>
+         Migrate hive table to a paimon table. Arguments:
+            <li>source_type: the origin table's type to be migrated, such as hive. Cannot be empty.</li>
+            <li>table: name of the origin table to be migrated. Cannot be empty.</li>
+            <li>options: the table options of the paimon table to migrate.</li>
+            <li>delete_origin: whether to delete the origin table metadata from hms after migrate. Default is true</li>
```

Comment:
I think we should mention the option of target_table in the description. Without target_table, this option is difficult to understand.

Reply:
Get, <li>target_table: name of the target paimon table to migrate. If not set would keep the same name with origin hive table</li> this describtion would be OK？